usr/bin/playtronos-session-select

```diff
@@ -0,0 +1,92 @@
+#! /bin/bash
+
+
+if [ $EUID -ne 0 ]; then
+	echo "$(basename $0) must be run as root"
+	exit 1
+fi
+
+
+SESSION_OVERRIDE="/etc/sddm.conf.d/60-playtron-session-override.conf"
+SESSION_LIST=('user' 'dev')
+SELECTED_SESSION="$1"
+
+if [ "$2" = "--no-switch" ]; then
+	NO_SWITCH=1
+fi
+
+function dev() {
+	mkdir -p $(dirname $SESSION_OVERRIDE)
+
+	echo '
+[Autologin]
+Session=playtron-weston
+' > ${SESSION_OVERRIDE}
+
+	# workaround for touch screen not working in dev session
+	systemctl stop inputplumber
```

Comment:
Disabling InputPlumber will break input testing. Can we investigate why you're seeing issues with the touchscreen?

Reply:
Yes, we should investigate. This is a temporary workaround. For the input test I thought we could start input plumber and stop it again afterwards.

Reply:
After some digging we found that updating inputplumber to 0.49.2 and restarting inputplumber during the session switch allowed the touch screen to function in the dev session.
The PR has been updated.